TailLight/TailLight.vcxproj

```diff
@@ -33,6 +33,7 @@
     <DriverType>KMDF</DriverType>
     <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>
     <ConfigurationType>Driver</ConfigurationType>
+    <Driver_SpectreMitigation>Spectre</Driver_SpectreMitigation>
```

Comment:
Why are you bundling Spectre-mitigation changes with the changes in this PR??  Please submit such change requests in a separate RP.